geode-core/src/main/java/org/apache/geode/management/DiskBackupStatus.java

```diff
@@ -14,91 +14,25 @@
  */
 package org.apache.geode.management;
 
-import java.util.HashMap;
 import java.util.Map;
-import java.util.Set;
-import java.util.stream.Collectors;
 
 import org.apache.geode.cache.persistence.PersistentID;
-import org.apache.geode.distributed.DistributedMember;
 
 /**
  * Composite data type used to distribute the status of disk backup operations.
  * 
  * @since GemFire 7.0
  */
-public class DiskBackupStatus {
-
-  /**
-   * Map of DistributedMember, Set<PersistentID>
-   */
-  private Map<String, String[]> backedUpDiskStores;
-
-  /**
-   * List of offline disk stores
-   */
-  private String[] offlineDiskStores;
-
+public interface DiskBackupStatus {
```

Comment:
Do you know if jmx composite types can be interfaces? If not it would be worth using jconsole to browse the DistributedSystemMXBean and invoked the backup op that returns this DiskBackupStatus.

Reply:
Checked in jconsole and confirmed that the interface form works. The standard MBean approach uses interface definitions with implementing classes.